charts/adguard-home/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v0.102.0
 description: DNS proxy as ad-blocker for local network
 name: adguard-home
-version: 2.1.1
+version: 2.1.2
```

Comment:
As there is a new feature being added, can you please bump the chart version to 2.2.0?

Reply:
yeah, makes sense. Thanks :)